packages/codemirror/style/base.css

```
@@ -237,10 +237,11 @@
 }
 
 .jp-CodeMirrorEditor .remote-caret > div {
+  white-space: nowrap;
   position: absolute;
-  top: -1.05em;
+  top: -1.1em;
```

Maybe add a little bit of padding at the bottom too, it will help against letters such as "y" getting cut out.

```suggestion
  top: -1.15em;
  padding-bottom: 0.05em;
```